client/src/pages/ContestDetails/ContestDetails.tsx

```diff
@@ -36,44 +45,54 @@ export default function ContestDetails(): JSX.Element {
   }
 
   return (
-    <Grid container component="main" className={classes.root} direction="column">
-      <CssBaseline />
-      <Grid container alignItems="center" justify="center">
-        <Grid item xs={12} sm={10} md={8}>
-          <Link to="/contest/all" className={classes.breadcrumb}>
-            <ArrowBackIosIcon fontSize="inherit" /> Back to contests list
-          </Link>
-        </Grid>
-        <Grid item xs={12} sm={10} md={8} className={classes.titleColumn}>
-          <Box display="flex" flexWrap="nowrap" alignItems="center" bgcolor="transparent">
-            <Box>
-              <Typography className={classes.contestTitle} component="h1" variant="h5">
-                {title}
-              </Typography>
-            </Box>
-            <Box flexGrow={1}>
-              <Button variant="contained" color="primary" disableElevation className={classes.prize}>
-                {prizeAmount}
-              </Button>
-            </Box>
-            <Box>
-              <Button variant="outlined" color="primary" className={classes.winnerButton}>
-                SELECT WINNER
-              </Button>
-            </Box>
-          </Box>
-          <Grid item xs={12} sm={10} md={8} className={classes.ownerColumn}>
-            <Box display="flex" alignItems="center">
-              <Box>
-                <Avatar src={demoProfilePhoto} alt="Profile Photo" />
+    <>
+      {submissionObj.length ? (
+        <Grid container component="main" className={classes.root} direction="column">
+          <CssBaseline />
+          <Grid container alignItems="center" justify="center">
+            <Grid item xs={12} sm={10} md={8}>
+              <Link to="/contest/all" className={classes.breadcrumb}>
+                <ArrowBackIosIcon fontSize="inherit" /> Back to contests list
+              </Link>
+            </Grid>
+            <Grid item xs={12} sm={10} md={8} className={classes.titleColumn}>
+              <Box display="flex" flexWrap="nowrap" alignItems="center" bgcolor="transparent">
+                <Box>
+                  <Typography className={classes.contestTitle} component="h1" variant="h5">
+                    {title}
+                  </Typography>
+                </Box>
+                <Box flexGrow={1}>
+                  <Button variant="contained" color="primary" disableElevation className={classes.prize}>
+                    {prizeAmount}
+                  </Button>
+                </Box>
+                <div>
+                  <Button
+                    component={Link}
+                    to={`/fileUpload/${contestId}`}
+                    variant="outlined"
+                    color="primary"
+                    className={classes.winnerButton}
+                  >
+                    SUBMIT DESIGN
```

Comment:
Please try to avoid using capital case letters when possible (certain screen readers pick them up as acronyms) , in this case you can use the class to transform the text to uppercase.

Reply:
noted